packages/salesforcedx-utils-vscode/test/unit/commands/progressNotification.test.ts

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import * as proxyquire from 'proxyquire';
+import { Subject } from 'rxjs/Subject';
+import * as sinon from 'sinon';
```

Comment:
Let's merge these two imports for `sinon`

Reply:
Updated - see https://github.com/forcedotcom/salesforcedx-vscode/pull/2660/commits/1b4a811e1eaf5e8e52575b8d550724bd598664ed